cmd/br/cmd.go

```diff
@@ -11,6 +11,8 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/pingcap/br/pkg/lightning/common"
```

Comment:
we should use [`github.com/pingcap/tidb-tools/pkg/utils`](https://github.com/pingcap/tidb-tools/blob/master/pkg/utils/security.go), or at least move this package out of Lightning.

Reply:
done